src/components/identity/lockup.js

```diff
@@ -1,20 +1,38 @@
 import React from 'react'
 import SbEditable from 'storyblok-react'
 import SbLink from "../partials/sbLink"
+import FullWidthImage from "../media/fullWidthImage";
 
 const Lockup = (props) => {
   return (
     <SbEditable content={props.blok}>
       <div className="su-lockup su-lockup--option-n">
         <SbLink link={props.blok.link}>
-          <div className="su-lockup__cell1">
-            <div className="su-lockup__wordmark-wrapper">
-              <span className="su-lockup__wordmark">Stanford</span>
-            </div>
-          </div>
-          <div className="su-lockup__cell2">
-            <span className="su-lockup__line1">{props.blok.lineOne}</span>
-          </div>
+          {props.blok.logoImage ? (
```

Comment:
```suggestion
          {props.blok.logoImage?.filename ? (
```
Storyblok image field is a bit finicky - I tested, without this change, it would't render the "Stanford" wordmark if there's no logo image (for the pre-existing hero banners, you won't see the "Stanford" wordmark going away until you click into the lockup component)
![Screen Shot 2021-08-19 at 4 23 37 PM](https://user-images.githubusercontent.com/42749717/130156520-eb572389-c6d9-4fda-8cf9-92e6c9d53947.png)

Reply:
Ooh. Good catch. Yeah I hadn't played around as much with how it behaves in the editor